app/helpers/Yaml.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace App\Helpers;
+
+use Symfony\Component\Yaml\Yaml as SymfonyYaml;
+use Symfony\Component\Yaml\Exception\RuntimeException;
+
+
+class YamlException extends \RuntimeException {}
+
+/**
+ * A Yaml wrapper to simplify transition from Symfony/Yaml to PECL yaml package.
+ * Historical note: we have switched to PECL since it is several orders of magnitude faster
+ * and performance become an issue when processing MiB-sized backend result yaml files.
+ * 
+ * Fallback to Symfony/Yaml is provided when extra formattin options are required.
```

Comment:
Typo: formattin

In addition, not really sure if fallback when options are provided is ideal... it is a bit hidden and not expected...